scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/ForkedJvmTaskExecutionCommandCreator.java

```diff
@@ -52,11 +51,14 @@
 
 
 public class ForkedJvmTaskExecutionCommandCreator implements Serializable {
-    private final static String javaHomePostfixJavaExecutable = File.separatorChar + "bin" + File.separatorChar +
-                                                                "java";
 
     private static final Logger logger = Logger.getLogger(ForkedJvmTaskExecutionCommandCreator.class);
 
+    private static final String JAVA_HOME_POSTFIX_JAVA_EXECUTABLE = File.separatorChar + "bin" + File.separatorChar +
+                                                                    "java";
+
+    private static final String JAVA_SECURITY_POLICY_FILE = "security.java.policy-client";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "JAVA_SECURITY_POLICY_FILE" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)